network-api/networkapi/wagtailpages/migrations/0052_buyersguideeditorialcontentindexpagearticlepagerelation.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.13 on 2022-08-24 17:45
```

Comment:
This migration is used to create the relationship between the Editorial Content Index page and its related articles